tgui/packages/tgui/interfaces/CompostBin.js

```diff
@@ -0,0 +1,96 @@
+import { useBackend, useSharedState } from '../backend';
+import {
+  Button,
+  Section,
+  Box,
+  Flex,
+  Icon,
+  Collapsible,
+  NumberInput,
+  ProgressBar,
+  Dimmer,
+  LabeledList,
+} from '../components';
+import { Window } from '../layouts';
+
+
+export const CompostBin = (props, context) => {
+	const { act, data } = useBackend(context);
+	const { biomass, compost, biomass_capacity, compost_capacity} = data;
+	
+	let [vendAmount, setVendAmount] = useSharedState(context, 'vendAmount', 1);
+	
+  return (
+    <Window>
+		<Window.Content>
```

Comment:
Can you run prettier on this please?
You can do this via the tasks in vscode with Ctrl+Shift+B and selecting "tgui: run prettier formatting"

Reply:
I'm using notepad++ for the js files, but for 2 lines I can just add the changes manually.

Reply:
Can you show me how it should look like?

Reply:
You should use VSCode for the js files too

Reply:
Theres a lot of stuff weird with the file, if you get/have vscode installed, you should should have access to the prettier vscode task. Its an automated change to the whole file

Reply:
Couldn't find a command related to tgui, but I did run prettier as is. Does it look fine?

Reply:
I ran "tgui: run prettier formatting" and it ran on the entire folder. It doesn't seem like it did anything prettier didn't though, at least to my own file, so I just discarded the changes.

Reply:
From the chat in discord, it seems like it should be fine so I'm resolving this.